.github/CONTRIBUTING.md

```diff
@@ -42,21 +42,32 @@ Gradle uses pull requests for contributions. Fork [gradle/gradle](https://github
     git config user.name 'First Last'
     git config user.email user@example.com
 
+### IntelliJ
+
 You can generate the IntelliJ project by running
 
-    ./gradlew idea
+    gradlew idea
+
+### Eclipse
```

Comment:
Let's write here that we welcome contributors using Eclipse but aren't able to guarantee a smooth development experience.